electronics_model/PriceGetter.py

```diff
@@ -0,0 +1,69 @@
+from typing import List, Tuple, Dict
+
+import edgir
+from edg_core import BaseBackend, CompiledDesign, TransformUtil
+import os
+import csv
+import re
+
+
+class GetPrice:
```

Comment:
Does this need a class? Can this just be a function in `GeneratePrice` - I don't think this needs any dedicated state?